pillar/edx/xqwatcher_600.sls

```diff
@@ -32,7 +33,7 @@ edx:
             CODEJAIL:
               name: mit-600x
               user: mit-600x
-              lang: python3
+              lang: {{ python3_version }}
```

Comment:
This needs to stay as `python3`. It's not a path attribute, it's an input to a selection list which only supports `python2` and `python3`